test/unit/org/apache/cassandra/io/sstable/SSTableReaderTest.java

```diff
@@ -143,6 +150,23 @@ public static void defineSchema() throws Exception
         CompactionManager.instance.disableAutoCompaction();
     }
 
+    @After
+    public void teardown()
+    {
+        for (Ref<?> ref : refsToRelease)
+        {
+            try
+            {
+                ref.release();
+            }
+            catch (Exception exc)
+            {
+                LOGGER.error("Error releasing ref during teardown: {}", ref, exc);
```

Comment:
This should fail the test not be caught. You can do the throwable merge pattern if you want to release all of them before failing the test and then throw the first one with the rest suppressed.